datascience-notebook/Dockerfile

```diff
@@ -10,11 +10,18 @@ ENV CHOWN_HOME=no
 
 COPY ./files/apt_packages /tmp/
```

Comment:
I know this isn't in the scope of this PR, but do you need to copy the file if you're just `cat`ing the contents to `apt-get install`?